Client/src/Components/Fallback/index.jsx

```diff
@@ -20,7 +20,7 @@ import "./index.css";
  * @returns {JSX.Element} The rendered fallback UI.
  */
 
-const Fallback = ({ title, checks, link = "/", isAdmin }) => {
+const Fallback = ({ title, checks, link = "/", isAdmin, ovalStart }) => {
```

Comment:
Hey Shemy, why is the name of this variable ovalStart?

Reply:
We could provid a fallback for that variable, since this component is already being called. You can set ovalStart  = false in the destructuring (I suggest changing for a more semantic name for the variable)

Reply:
@shemy-gan was this meant to be `vowelStart`?  That makes sense?  In the interest of expediencey I'll push that change to the branch, we can revisit if there was a special meaning for `ovalStart`